pipeline-runner/init.R

```diff
@@ -28,10 +28,16 @@ buildActivityArn <- function(aws_region, aws_account_id, activity_id) {
 }
 
 load_config <- function(development_aws_server, api_version = "v1") {
-    label_path <- "/etc/podinfo/labels"
-    aws_account_id <- Sys.getenv("AWS_ACCOUNT_ID", unset="242905224710")
-    aws_region <- Sys.getenv("AWS_DEFAULT_REGION", unset="eu-west-1")
+    tryCatchLog({
+        aws_account_id <- Sys.getenv("AWS_ACCOUNT_ID")
+        aws_region <- Sys.getenv("AWS_DEFAULT_REGION")
```

Comment:
if the account id or the region is not defined, we want to fail immediately. Right now in master, if they are not defined, they will be set to our account information, which will make the pipeline "seem to work". This is bad, because we will realise there is a problem much later after it occurred, making it difficult to debug